submissions/AbrahamJudah/HTML&CSS/SignUp.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Registration Form</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css">
+</head>
+<body>
+    <section class="container">
+        <img src="Pics/logo.png" alt="logo" width="170" height="80" class="logo"> 
+        <div class="registration-block">
+            <form action="" method="POST">
+                <h3 class="registration-title">Create Your Account</h3>
+                <label for="first name" class="registration-label">First Name</label>
```

Comment:
Well done Chigozie
Attribute values like name, id, etc., should not have spaces. You can separate words with a hyphen or underscore. Please fix all ocurrences